gradle/dependencies.gradle

```diff
@@ -9,16 +9,17 @@ ext {
   supportLibrariesVersion = '26.0.1'
 
   dep = [
-      androidPlugin      : 'com.android.tools.build:gradle:3.0.1',
-      okhttp             : "com.squareup.okhttp3:okhttp:$okhttpVersion",
-      mockWebServer      : "com.squareup.okhttp3:mockwebserver:$okhttpVersion",
-      pollexor           : 'com.squareup:pollexor:2.0.4',
-      supportV4          : "com.android.support:support-v4:$supportLibrariesVersion",
-      supportAnnotations : "com.android.support:support-annotations:$supportLibrariesVersion",
-      junit              : 'junit:junit:4.12',
-      truth              : 'com.google.truth:truth:0.36',
-      robolectric        : 'org.robolectric:robolectric:3.1',
-      mockito            : 'org.mockito:mockito-core:1.9.5'
+      androidPlugin        : 'com.android.tools.build:gradle:3.0.1',
+      okhttp               : "com.squareup.okhttp3:okhttp:$okhttpVersion",
+      mockWebServer        : "com.squareup.okhttp3:mockwebserver:$okhttpVersion",
+      pollexor             : 'com.squareup:pollexor:2.0.4',
+      supportV4            : "com.android.support:support-v4:$supportLibrariesVersion",
+      supportAnnotations   : "com.android.support:support-annotations:$supportLibrariesVersion",
+      supportExifInterface : "com.android.support:exifinterface:$supportLibrariesVersion",
+      junit                : 'junit:junit:4.12',
+      truth                : 'com.google.truth:truth:0.36',
+      robolectric          : 'org.robolectric:robolectric:3.1',
+      mockito              : 'org.mockito:mockito-core:1.9.5'
```

Comment:
Who thought column alignment was a good idea 🤦‍♂️ 

(Don't let it be me... don't let it be me...)